WBooks/Views/Library/LibraryViewController.swift

```diff
@@ -84,4 +74,33 @@ extension LibraryViewController: UITableViewDataSource, UITableViewDelegate {
         }
         return cell
     }
+    
+    //Cuando se seleccione alguna de las columas, redirigir de vista.
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        // Guarda el book instance del libro de la fila seleccionada.
+        let book = _viewModel.books[indexPath.row]
+        //********let bookDetailViewModel = BookDetailViewModel(bookVM: book)
+        //********let bookDetailViewController = BookDetailViewController(viewModel: bookDetailViewModel)
+        //********let bookDetailViewController = BookDetailViewController(viewModel: book)
```

Comment:
These comments shouldn't be in the final version for a PR, as they already were committed to the branch previously and will remain there.